PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutLineItem.java

```diff
@@ -137,6 +159,33 @@ public void setUrl(@NonNull String url) {
         this.url = url;
     }
 
+    /**
+     * The image URL to product information.
+     *
+     * @param imageUrl The image URL with additional information.
+     */
+    public void setImageUrl(@NonNull String imageUrl) {
+        this.imageUrl = imageUrl;
+    }
+
+    /**
+     * UPC code of the item.
+     *
+     * @param upcCode The UPC code.
+     */
+    public void setUpcCode(@NonNull String upcCode) {
+        this.upcCode = upcCode;
+    }
+
+    /**
+     * UPC type of the item.
+     *
+     * @param upcType The UPC type.
+     */
+    public void setUpcType(@NonNull String upcType) {
```

Comment:
```suggestion
    public void setUpcType(@NonNull @PayPalLineItemUpcType String upcType) {
```

This type should be added here and in the getter right (and in doc string too probably)? Or is that interface used somewhere else?

Reply:
Added interface type to setter and getter annotations